scripts/validate_docstrings.py

```diff
@@ -125,18 +169,22 @@ def needs_summary(self):
 
     @property
     def doc_parameters(self):
-        return self.doc['Parameters']
+        return {name: (type_, ''.join(desc))
+                for name, type_, desc in self.doc['Parameters']}
```

Comment:
best to use OrderedDict for people not on python >3.6, otherwise the order of the params is lost (and you get incorrect error messages about order not matching)